pages/p/[productId].js

```diff
@@ -72,7 +84,7 @@ const Product = React.memo(lazyProps => {
   const classes = useStyles()
   const product = get(state, 'pageData.product') || {}
   const color = get(state, 'pageData.color', {})
-  const size = get(state, 'pageData.size')
+  const size = get(state, 'pageData.size', {})
```

Comment:
Lodash does some weird stuff here with getting the default value in which the null somehow slips by sometimes. So it's better to use just get(.., ..) || {} for all those 3.